app/services/repo_label_assigner.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+# PORO
+# This class takes in a repo and fetch all labels for that repo. It then
+# creates labels and associates them with the passed in repo
+#
+# Example:
+#
+#   repo = Repo.first
+#   assigner = RepoLabelAssigner.new(repo: repo)
+#   assigner.create_and_associate_labels!
+#
+class RepoLabelAssigner
+  def initialize(repo:)
+    @repo = repo
+    url = ['repos', repo.user_name, repo.name, 'labels'].join('/')
+    @github_bub_response = GitHubBub.get(url)
+  end
+
+  def create_and_associate_labels!
+    return unless github_bub_response.success?
+
+    remote_labels.each do |label_hash|
+      label_name = label_hash['name'].downcase
```

Comment:
The issue stated that we wanted to only store `hacktoberfest` related labels. I'm happy to add a `next unless` line here if that's still the case but this is pretty simple as is.